l10n_eu_nace/migrations/12.0.2.0.0/post-migration.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 ACSONE SA/NV
+# Copyright 2020 Tecnativa - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+def migrate(cr, version):
+    cr.execute(
+        """
+            UPDATE res_partner rp
+            SET nace_id = imd2.res_id
+            FROM ir_model_data imd1
+            LEFT JOIN ir_model_data imd2
+                ON  imd2.module = imd1.module
+                AND imd2.name = substring(imd1.name, 4)
```

Comment:
Oops, a little big bug here:

```suggestion
                AND imd2.name = substring(imd1.name, 5)
```

FTR:

```sql
devel=# select substring('old_hi', 4);
 substring 
-----------
 _hi
(1 row)

devel=# select substring('old_hi', 5);
 substring 
-----------
 hi
(1 row)
```